controller/TestCenter.php

```diff
@@ -79,7 +79,7 @@ public function testCenter()
             $this->composeView(
                 'testcenters-testcenter',
                 $data,
-                'pages/index.tpl',
+                'TestCenter/index.tpl',
```

Comment:
I guess we'd better have a tech debt task in order to prevent this from happening in the future.

Not like I have a good suggestion on how to do it, though :P

Reply:
Indeed, it will happen all occasions where the template path is collides with a tao-core path.
`pages/index.tpl` sounds like a template path used on another extensions